src/v2/view-builder/views/email/BaseAuthenticatorEmailView.js

```diff
@@ -53,6 +53,7 @@ const Body = BaseFormWithPolling.extend(Object.assign(
     },
     initialize() {
       BaseFormWithPolling.prototype.initialize.apply(this, arguments);
+      this.formSubmitted = false;
```

Comment:
instead of controlling the form state.. before next poll request the error should be cleared.

Reply:
@pradeepdewda-okta 
I don't think we should clear the error before the next poll since the next poll could occur in a few seconds which mean the error would stay on for only a few seconds.
Polling should be somethings in a background so I think we should keep it in the form state so things only get update when user interact with the UI rather than update the UI based on something in the background.

Reply:
There is nothing like background polling. Widget polls if there is a poll remediation, if poll contains a form error then API should send a form error with remediation ie. [ remediation + error ]..

So what i am suggesting is in this particular case next poll happens in 60 sec then the error will remain there for 60sec and as soon as we get the remediation from next poll, then new error or state should be rendered..

Reply:
@mauriciocastillosilva-okta @jmelberg-okta can you guys please help review this PR ??

Reply:
@pradeepdewda-okta 
If we clear error before polling in this case, then error would only stay on till next poll which is 60 sec. But if the next poll has no error, shouldn't the error stay on longer?

Reply:
if next response contains an error then ideally widget should render the error again...

Reply:
I mean if the next polling response contains no error, shouldn't the previous error stay (if it is from user interaction by submitting OTP)?

Reply:
Ideally no. new remediation ==> new screen (UI).

Reply:
I am not entirely following: why are we polling if there is a rate limit error? iIf we get a 429 response, there is no remediation that says we should keep polling, so we should stop there and show the error message we have that says "Try again later" and stop any other processing.

Reply:
@mauriciocastillosilva-okta 
Current behaviour is that if poling encounter rate limit error, SIW should:
1. Stop polling and wait 60 secs before start polling again (60 sec should be enough time for rate limit to reset)
2. Hide rate limit error if polling encounter that since it being handled in number 1.

The bug is that current logic doesn't distinguish rate limit error between when user OTP submission (saving form) and when polling. So rate limit error is always hidden to user during OTP submission. 

This logic should help with that. Note that `triggerAfterError(model, error)` isn't aware of what is causing error (user vs polling), so the solution is tracking of it in the form state. (Rename `isPollingRateLimitError` to `isRateLimitError` since it a little misleading).So if the user submit the OTP and encounter rate limit error, the error should stay on and the polling would pause for 60 sec to give time for rate limit to reset.

Reply:
@worldleelavivat-okta the behavior to resume polling after 1 min should be dictated by API response, Widget is NOT going to handle such one offs.. with JWE token introduction where stateToken can change between any two remediation this is going to break.

Reply:
@pradeepdewda-okta 
While I agree as it should be handled more gracefully by API response in the future when JWE token is introduced. This bug fix is for `OIE-Tuscany` so can we leave that as part of future enhancement as part of JWE token introduction as it would change the current behaviour that was signed off from Product before?

Reply:
@worldleelavivat-okta Well it's not JWE token specific. JWE token actually magnifies the issue with this implementation and a genuine use case where stateToken changes.. Currently this flow works because there is no change in stateToken, but you should not assume that stateToken is persistent even in current idx implementation.. 

See if you can change this to `terminal` error state, if changing API is not possible.. `Resuming` polling without IDX remediation is a NO NO.

Reply:
For reference, discussion in: https://okta.slack.com/archives/C0131CX1MGB/p1625683893070300